### DISCUSSION TITLE
blog/2022/221115-issue-of-position-or-right-and-wrong/

### DISCUSSION THREAD

**Discussion by giscus** [zhanbao2000/blog#16] [Announcements]
# blog/2022/221115-issue-of-position-or-right-and-wrong/

很多事情其实是根本无法评论对错的，这类问题绝大多数和立场有关，叫做“立场问题”。什么是立场问题？你站在哪边为谁说话，就是坚持谁的立场，你就会认为谁是对的。这类问题实际上和谁对谁错没关系，更重要的是你为谁说话。如果不需要为任何一个人说话，仍然能判断对错或者得到一个二元的判断，那叫做“是非问题”。

https://blog.arisa.moe/blog/2022/221115-issue-of-position-or-right-and-wrong/

**Comment by kirito-ui** [comment]
受教了
